streaming/java/streaming-state/src/main/java/org/ray/streaming/state/keystate/state/impl/StateHelper.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ray.streaming.state.keystate.state.impl;
+
+import com.google.common.base.Preconditions;
+import org.ray.streaming.state.backend.TransactionKeyStateBackend;
+import org.ray.streaming.state.keystate.desc.AbstractStateDescriptor;
+
+/**
+ * State Helper Class.
+ */
+public class StateHelper<T> {
```

Comment:
Can we have a better name? Helper is more like an util which mostly methods are static